src/containers/Main/Main.js

```diff
@@ -1,53 +1,51 @@
-import { getAll } from '../../actions/recipes.actions';
-import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { Layout, Spin } from 'antd';
-import RecipeCard from '../../components/RecipeCard';
-import './Main.css';
+import { getAll } from "../../actions/recipes.actions";
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Layout, Spin } from "antd";
+import RecipeCard from "../../components/RecipeCard";
+import "./Main.css";
```

Comment:
![](https://media.giphy.com/media/yr7n0u3qzO9nG/giphy.gif)

Reply:
ahhaahahha  I don't know what happened. I have a linter that's crazy